networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/libp2p/gossip/GossipHandler.java

```diff
@@ -82,7 +82,8 @@ public void gossip(Bytes bytes) {
     }
 
     LOG.trace("Gossiping {}: {} bytes", topic, bytes.size());
-    SafeFuture.of(publisher.publish(Unpooled.wrappedBuffer(bytes.toArrayUnsafe()), topic))
+    SafeFuture.of(
+            publisher.publishExt(Unpooled.wrappedBuffer(bytes.toArrayUnsafe()), null, null, topic))
```

Comment:
Looks like these 2 extra params default to `null`, but I guess you just want to be explicit here?

Reply:
Oh, right! 👍 
Will revert this then